src/Aquifer.Public.API/OpenApi/ClientGenerationSettings.cs

```diff
@@ -0,0 +1,202 @@
+﻿using FastEndpoints.ClientGen.Kiota;
+using Kiota.Builder;
+
+namespace Aquifer.Public.API.OpenApi;
+
+/// <summary>
+/// Generate client code and make it available as a zip file per language at `/clients/language-id`.
+/// See https://fast-endpoints.com/docs/swagger-support#api-client-generation.
+/// We could also generate clients from the command line or on build if desired.
+/// </summary>
+public static class ClientGenerationSettings
+{
+    private const string _clientsRouteName = "clients";
+
+    public static IEndpointRouteBuilder ConfigureClientGeneration(
+        this IEndpointRouteBuilder app,
+        string swaggerDocumentName,
+        TimeSpan clientZipCacheDuration)
+    {
+        app.MapApiClientEndpoint(
+            $"/{_clientsRouteName}/cs",
+            c =>
+            {
+                c.SwaggerDocumentName = swaggerDocumentName;
+                c.Language = GenerationLanguage.CSharp;
+                c.ClientNamespaceName = "BiblioNexus.Aquifer.API.Client";
+                c.ClientClassName = "AquiferClient";
+                c.OutputPath = Path.Combine(c.OutputPath, "cs", Path.GetRandomFileName());
```

Comment:
I think there's a bug in FastEndpoints where if you specify `CleanOutput = true` the client generation blows up if you use a non-default `OutputPath`.  I worked around this by just using a different temp folder every time we generate a client (which never gets erased until the Temp folder is cleared by the OS).